### ISSUE TITLE
Demo doesn't work in eu-west-1

### ISSUE THREAD

**Issue by rogerchi** [binxio/cfn-secret-provider#3]
When attempting to run the demo on eu-west-1, the stacks hang on creation and I get this error in the lambda function binxio-cfn-secret-provider:

failed to put the response to https://cloudformation-custom-resource-response-euwest1.s3-eu-west-1.amazonaws.com/ [...]
<Error><Code>SignatureDoesNotMatch</Code><Message>The request signature we calculated does not match the signature you provided. Check your key and signing method.</Message>

**Comment by mvanholsteijn** [comment]
I see this problem too. It is really strange, as the responseURL is passed in by CloudFormation. We just execute the PUT. I will try to open a case.

**Comment by wdegeus** [comment]
I ran into exactly the same problem, which seems to be fixed after adding a {'Content-Type': ''} header to the requests.put() request (which I saw other people do).
In cfn_resource_provider.py around line 360:
```        
        headers = {'Content-type': ''}
        r = requests.put(url, json=self.response, headers=headers)
```

**Comment by mvanholsteijn** [comment]
Solved by #7cf6a2c863d6acc2318159aef5c67c4800892b87 released in version 0.7.4
